sfm_pc/sql/memberships.sql

```diff
@@ -54,6 +54,8 @@ LEFT JOIN organization_organization AS member
   ON member_organization.value_id = member.id
 LEFT JOIN organization_organizationname AS member_name
   ON member.id = member_name.object_ref_id
+JOIN organization_organizationdivisionid as member_country
```

Comment:
I have this as a JOIN instead of a LEFT JOIN to mirror the format earlier in the file. I'd love to talk about this irl for a few minutes to understand the complexities and possible implications of this. (I don't anticipate any negative consequences.)

Reply:
This is a great point to raise. Unfortunately our use of `JOIN` and `LEFT JOIN` isn't entirely consistent in this file as far as I can tell, so it's hard to know when it's appropriate to copy/paste.

As a very general rule of thumb, `LEFT JOIN` is useful when you want all rows from the "left" table, regardless of whether the row has a match in the "right" table. (A regular `JOIN`, which is shorthand for `INNER JOIN`, will discard any rows in the left table that don't have a match in the right table.) [This SO answer](https://stackoverflow.com/a/3308153) provides a really nice visual explanation of the difference. In practice, this often means that `LEFT JOIN` is useful if the field you're joining on can be null, but you want all results anyway.

In the joins in this file, `LEFT JOIN` predominates, because the majority of these fields are optional but we typically want all results anyway in order to represent the full set of memberships. Two exceptions are the `JOIN` on `organization_organizationname`, which is a required field for `Organizations`, and the `JOIN` on `membershiporganization_membershiporganization`, which makes sense because we don't want any results for organizations that don't have memberships.

This all being said, I believe that line 22 where we `JOIN` on `organization_organizationdivisionid` is actually a mistake, and should be a `LEFT JOIN`. Looking at the `Organization` model, `divisionid` isn't actually a required attribute:

https://github.com/security-force-monitor/sfm-cms/blob/886fba9fdf1271178115f18b6f36bc8e76f5aa28/organization/models.py#L66-L68

In practice, all the `Organizations` in our database currently have a value for `divisionid`, which means that the two joins won't produce different results. However, a user could feasibly create an `Organization` without a `divisionid` in the future. For this reason, I think the right call is to perform this join as a `LEFT JOIN`. Ultimately it would be good to update the previous `JOIN` to match this behavior, but for now I'm fine just scoping this to the line you added in this PR.

Reply:
Thank you for the detailed explanation—particularly the relation of the type of `JOIN` to the required fields. 

Unfortunately, all of the sql files have a regular JOIN on the `organization_organizationdivisionid` table.